_about/notes/0.5.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Istio 0.5
+
+order: 96
+
+layout: about
+type: markdown
+---
+{% include home.html %}
+
+In addition to the usual pile of bug fixes and perf improvements, this release includes the new or 
+updated features detailed below.
+
+## Networking
+
+- **Control Plane Deployments**. `istioctl` now has a deployment command to produce istio control plane deployments. TODO: Doc link?
+
+- **Automatic Proxy Injection**. Use Kubernetes 1.9's mutating webhook functionality to provide automatic
+pod-level proxy injection. Automatic injection requires Kubernetes 1.9 or beyond and
+therefore doesn't work on older versions. [Learn more]({{home}}/docs/setup/kubernetes/sidecar-injection.html#automatic-sidecar-injection)
+
+- **Alpha2 Traffic Rules**. We've started to add support for a new model for routing and destination rules. You can see the updated
+definitions [here]({{home}}/docs/reference/config/istio.routing.v1alpha2.html). It's not quite ready to be used though, this is
+still very much a work in progress.
+
+## Mixer adapters
+
+- **Open Policy Agent**. Mixer now has an authorization adapter implementing the [open policy agent](http://www.openpolicyagent.org) model,
+providing a flexible fine-grained access control mechanism. TODO: Doc link?
```

Comment:
@wattli Do we have a doc link for this?

Reply:
https://docs.google.com/document/d/1U2XFmah7tYdmC5lWkk3D43VMAAQ0xkBatKmohf90ICA/edit#heading=h.fmlgl8m03gfy